bftengine/src/bftengine/BftCommandHandler.cpp

```diff
@@ -0,0 +1,47 @@
+#include <BftCommandHandler.h>
+#include <KeyManager.h>
+#include <sstream>
+
+namespace bftEngine {
+
+int BftCommandHandler::execute(uint16_t clientId,
+                               uint64_t sequenceNum,
+                               uint8_t flags,
+                               uint32_t requestSize,
+                               const char *request,
+                               uint32_t maxReplySize,
+                               char *outReply,
+                               uint32_t &outActualReplySize,
+                               uint32_t &outReplicaSpecificInfoSize,
+                               concordUtils::SpanWrapper &parent_span) {
+  if (flags & KEY_EXCHANGE_FLAG) {
+    KeyManager::KeyExchangeMsg ke = KeyManager::KeyExchangeMsg::MsgDeserializer(request, requestSize);
```

Comment:
Usually, functions and method' names are verbs and classes/structs are nouns.
Any particular reason for this?
Can we call it `deserialize` -- it is a member of `KeyExchangeMsg` and obviously returns a message.

Reply:
deserialize is taken from the Serializable interface, otherwise, I would use it

Reply:
I see but what is the problem to call it the same name?
The method will just call the parent class' method `SerializableFactory::deserialize`.

Reply:
It means that every usage of the `deserialize` interface method within that class, needs to be prefixed with `Serializable:: ` .
It's error prone and too verbose IMO

Reply:
I'm calling it `deserializeMsg` .

Reply:
If the verbosity/complexity is hidden from the class user, then it is fine.
Anyways, I am ok with `deserializeMsg`.